test/fuzz/simplify.cpp

```diff
@@ -95,13 +93,13 @@ Expr random_condition(Type T, int depth, bool maybe_scalar) {
     };
     const int op_count = sizeof(make_bin_op) / sizeof(make_bin_op[0]);
 
-    if (maybe_scalar && rng() % T.lanes() == 0) {
+    if (maybe_scalar && fdp.ConsumeBool()) {
         T = T.element_of();
     }
 
-    Expr a = random_expr(T, depth);
-    Expr b = random_expr(T, depth);
-    int op = rng() % op_count;
+    Expr a = random_expr(fdp, T, depth);
+    Expr b = random_expr(fdp, T, depth);
+    int op = fdp.ConsumeIntegralInRange<int>(0, op_count);
```

Comment:
Should be `fdp.ConsumeIntegralInRange<int>(0, op_count-1)` as per #7546

Reply:
or better yet, use PickValueInArray()?

Reply:
True that would be better